### PR TITLE
603 update sbt

### LINKED ISSUE #603: Update sbt to latest version 1.8.2
It was reverted due to problems with example applications startup. When updating check if the applications do work and if not - fix them.

### PR DESCRIPTION
**Pull request by hwgonz**
Closes #603 
___

## Summary:

Fix an execution bug for running examples with sbt 1.8.0 or higher

I think this change in Fork behavior is the reason for this issue. 
https://github.com/sbt/sbt/commit/ca45370ea18463c27135293f35856ea6f070c0d0

So I am filtering out line.separator Java option as a solution

### CODE REVIEW

**Comment by ptrdom** [comment]
Would be neat to get https://github.com/ScalaConsultants/mesmer/pull/612 merged first, then we could use the updated CI to validate this PR.

**Review by lgajowy — APPROVED** [review]
I Agree with @ptrdom - let's merge #612 first to validate the CI job

**Comment by lgajowy** [comment]
@hwgonz I merged https://github.com/ScalaConsultants/mesmer/pull/612 that would test this PR automatically. Could you resolve the conflicts? Is there a chance that with the current project structure, the fix will not be required (that's my long shot ;) )?

File: otel-extension/src/it/scala/io/scalac/mesmer/instrumentation/akka/actor/otel/AkkaActorTest.scala [hunk 1]
```diff
@@ -196,7 +196,7 @@ final class AkkaActorTest
   }
 
   it should "record mailbox size properly" in {
-    val processingTime = 200
+    val processingTime = 1000
```
Comment (ptrdom) [inline otel-extension/src/it/scala/io/scalac/mesmer/instrumentation/akka/actor/otel/AkkaActorTest.scala:199]: I introduced this change too in https://github.com/ScalaConsultants/mesmer/pull/616, I think we can skip it here and keep it to a version bump only.

**Review by ptrdom — CHANGES_REQUESTED** [review]
Minor suggestion.

File: otel-extension/src/it/scala/io/scalac/mesmer/instrumentation/akka/actor/otel/AkkaActorTest.scala (hunk 1 above)
Comment (hwgonz) [reply otel-extension/src/it/scala/io/scalac/mesmer/instrumentation/akka/actor/otel/AkkaActorTest.scala:199]: @ptrdom Sure, the problem is that if I don't increase this time, the Windows build fails and won't allow me to merge the PR

Comment (ptrdom) [reply otel-extension/src/it/scala/io/scalac/mesmer/instrumentation/akka/actor/otel/AkkaActorTest.scala:199]: Ah, damn 🤦 @lgajowy what should we do?

Comment (lgajowy) [reply otel-extension/src/it/scala/io/scalac/mesmer/instrumentation/akka/actor/otel/AkkaActorTest.scala:199]: I knew that it's going to be a flaky test! ;/ Let's merge this PR and then let's fix anything downstream

**Review by lgajowy — APPROVED** [review]

**Pull request merged**